SETUP.md

```diff
@@ -0,0 +1,22 @@
+# Setting Up the App
+
+## First Steps
```

Comment:
Could you add a subsection here on the system-level dependencies people will need? (i.e. node, python 3, pipenv, npm, etc.)